cluster-autoscaler/core/scale_up.go

```diff
@@ -295,6 +295,17 @@ func executeScaleUp(context *AutoscalingContext, info nodegroupset.ScaleUpInfo)
 }
 
 func addAutoprovisionedCandidates(context *AutoscalingContext, nodeGroups []cloudprovider.NodeGroup, unschedulablePods []*apiv1.Pod) {
+	autoprovisionedNodeGroupCount := 0
+	for _, group := range nodeGroups {
+		if group.Autoprovisioned() {
+			autoprovisionedNodeGroupCount++
+		}
+	}
+	if autoprovisionedNodeGroupCount >= context.AutoscalingOptions.MaxAutoprovisionedNodeGroupCount {
```

Comment:
nit: I don't think we use "context.AutoscalingOptions" anywhere else in the code, is it possible to use just "context.MaxAutoprovisionedNodeGroupCount" here?

Reply:
Yes, fixed.